systemtests/src/main/java/com/github/streamshub/systemtests/setup/StrimziOperatorSetup.java

```diff
@@ -0,0 +1,306 @@
+        RESOURCE_MANAGER.deleteResource(getBundleClusterRoleBindings());
+        RESOURCE_MANAGER.deleteResource(getBundleConfigMap());
+        RESOURCE_MANAGER.deleteResource(getBundleDeployment());
+    }
+
+    private ClusterRoleBinding[] getStrimziWatchAllCrbs() {
+        return new ClusterRoleBinding[]{
+            new ClusterRoleBindingBuilder()
+                .withNewMetadata()
+                    .withName(Constants.STRIMZI_CO_NAME + "-namespaced")
+                .endMetadata()
+                .withNewRoleRef()
+                    .withApiGroup(Constants.RBAC_AUTH_API_GROUP)
+                    .withKind(ResourceKinds.CLUSTER_ROLE)
+                    .withName(Constants.STRIMZI_CO_NAME + "-namespaced")
+                .endRoleRef()
+                .withSubjects(new SubjectBuilder()
+                    .withKind(ResourceKinds.SERVICE_ACCOUNT)
+                    .withName(Constants.STRIMZI_CO_NAME)
+                    .withNamespace(this.deploymentNamespace)
+                    .build())
+                .build(),
+            new ClusterRoleBindingBuilder()
+                .withNewMetadata()
+                    .withName(Constants.STRIMZI_CO_NAME  + "-entity-operator")
+                .endMetadata()
+                .withNewRoleRef()
+                    .withApiGroup(Constants.RBAC_AUTH_API_GROUP)
+                    .withKind(ResourceKinds.CLUSTER_ROLE)
+                    .withName(Constants.STRIMZI_CO_NAME  + "-entity-operator")
+                .endRoleRef()
+                .withSubjects(new SubjectBuilder()
+                    .withKind(ResourceKinds.SERVICE_ACCOUNT)
+                    .withName(Constants.STRIMZI_CO_NAME)
+                    .withNamespace(this.deploymentNamespace)
+                    .build())
+                .build(),
+            new ClusterRoleBindingBuilder()
+                .withNewMetadata()
+                    .withName(Constants.STRIMZI_CO_NAME + "-watched")
+                .endMetadata()
+                .withNewRoleRef()
+                    .withApiGroup(Constants.RBAC_AUTH_API_GROUP)
+                    .withKind(ResourceKinds.CLUSTER_ROLE)
+                    .withName(Constants.STRIMZI_CO_NAME + "-watched")
+                .endRoleRef()
+                .withSubjects(new SubjectBuilder()
+                    .withKind(ResourceKinds.SERVICE_ACCOUNT)
+                    .withName(Constants.STRIMZI_CO_NAME)
+                    .withNamespace(this.deploymentNamespace)
+                    .build())
+                .build()
+        };
+    }
+
+    private OperatorGroup getStrimziOperatorGroup() {
+        return new OperatorGroupBuilder()
+            .editOrNewMetadata()
+                .withName(Constants.CONSOLE_OPERATOR_GROUP_NAME)
+                .withNamespace(this.deploymentNamespace)
+                .withLabels(Collections.singletonMap(Labels.APP, Constants.STRIMZI_NAME))
+            .endMetadata()
+            .build();
+    }
```

Comment:
Is the `OperatorGroup` necessary when installing without OLM?

Reply:
Had to check, but it seems like i should remove it from here. Thanks!